app/services/discovery_engine/sync/delete.rb

```diff
@@ -8,7 +8,7 @@ def initialize(
       content_id = nil, payload_version: nil,
       client: ::Google::Cloud::DiscoveryEngine.document_service(version: :v1)
     )
-      super()
+      super(content_id, payload_version:)
 
       @content_id = content_id
       @payload_version = payload_version
```

Comment:
`@content_id` and `@payload_version` are being assigned in the Operation superclass so the assignment should be removed from here.
I think this was missed because a similar assigned was removed from the Put class.

Reply:
Oops 🙈 Good spot!